content/user/running/local_testing.md

```diff
@@ -39,6 +39,19 @@ You will need to go through this file and set each entry to point to your local
 ## 4. Try a simple update to see if things work. 
 Try updating (for example), `includes/languages/YOURTEMPLATE/english.php` and, then refresh your browser page and make sure the change appears. 
 
+Remember that `*AMP` allows you to switch between versions of PHP.  So you can test your live site while your upgrade is being done, and test your upgrade as well simply by switching PHP versions.
+
+### Things to consider: 
+
+- Email sent by a local installation will often be discarded as spam by real mail servers.  If you need to view emails created by your local test cart, some options are: 
+  - using [MailHog](https://github.com/mailhog/MailHog) or a similar test tool.  Here are the [instructions from MAMP on using MailHog](https://documentation.mamp.info/en/MAMP-PRO-Mac/Servers-and-Services/MailHog/).
```

Comment:
Mailtrap.io is also easy to configure, and free tier is enough.

Reply:
added.